### PR TITLE
Add postgres client/extensions to images

### PR DESCRIPTION
**Pull request by andytson-inviqa**

### CODE REVIEW

**Comment by andytson-inviqa** [comment]
approved - I can't approve the PR even if none of it is my work now

**Comment by kierenevans** [comment]
Thanks Andy, still your work too :)

**Review by kierenevans — DISMISSED** [review]

**Review by kierenevans — APPROVED** [review]

**Pull request merged**
